packages/manager/apps/zimbra/public/translations/dashboard/Messages_fr_FR.json

```diff
@@ -1,6 +1,7 @@
 {
   "zimbra_dashboard_administrator_guide": "Guide administrateur",
   "zimbra_dashboard_add-domain": "Ajouter un domaine",
+  "zimbra_dashboard_delete-domain": "Supprimer un domaine",
```

Comment:
```suggestion
  "zimbra_dashboard_domains_delete": "Supprimer un domaine",
```

Reply:
There is some change on breadcrumb 
Can you target feat/MANAGER-13884 or fix/MANAGER-14501